tests/unit/claim-deduction.test.js

```diff
@@ -1,6 +1,5 @@
 import { Ed25519KeyPair, suites } from 'jsonld-signatures';
```

Comment:
The changes in this document are only because I pulled the `documentLoader` out of this file and put it in `tests/cached-document-loader.js`.